main.css

```diff
@@ -1299,8 +1301,56 @@ footer ul .aTopMargin {
   border-radius: 4px;
   text-decoration: none;
   align-self: flex-end;
-  color: #FFFFFF;
+  color: #ffffff;
   text-decoration: none;
 }
 
+
+
+.theStoryContainer {
+  display: grid;
+  grid: repeat(3);
+  grid-template-columns: repeat(auto-fit, minmax(33.33%, 1fr));
+  margin-right: auto;
+  margin-left: auto;
+}
+
+.firstImg {
+  grid-column: 1;
+  grid-row: 1;
+  width: 100%;
+  height: auto;
+}
+
+.history {
+  grid-column: 2;
+  grid-row: 1;
+  font-size: 1.2rem;
+}
+
+.secondImg {
+  grid-column: 2;
+  grid-row: 2;
+  width: 100%;
+  height: auto;
+}
+
+.vision {
+  grid-column: 1;
+  grid-row: 2;
+  font-size: 1.2rem;
+}
+
+@media screen and (min-width: 800px) {
+  .theStoryContainer {
+    width: 100%;
+    max-width: 1280px;
+    height: auto;
+    object-fit: cover;
+    align-content: center;
+    align-items: center;
+    align-self: center;
+    text-align: center;
+  }
```

Comment:
I think it looks cleaner if ipsum lorem text and its heading  is text-align: start instead of center both in smaller and larger screens.